form-view-model/src/test/kotlin/com/ritense/formviewmodel/service/FormViewModelServiceTest.kt

```diff
@@ -0,0 +1,125 @@
+package com.ritense.formviewmodel.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.ritense.authorization.AuthorizationService
+import com.ritense.formviewmodel.BaseTest
+import com.ritense.formviewmodel.viewmodel.TestViewModel
+import com.ritense.formviewmodel.viewmodel.TestViewModelLoader
+import com.ritense.formviewmodel.viewmodel.ViewModel
+import com.ritense.formviewmodel.viewmodel.ViewModelLoaderFactory
+import com.ritense.valtimo.camunda.domain.CamundaTask
+import com.ritense.valtimo.contract.json.MapperSingleton
+import com.ritense.valtimo.service.CamundaTaskService
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class FormViewModelServiceTest : BaseTest() {
+
+    private lateinit var objectMapper: ObjectMapper
+    private lateinit var viewModelLoaderFactory: ViewModelLoaderFactory
+    private lateinit var camundaTaskService: CamundaTaskService
+    private lateinit var authorizationService: AuthorizationService
+    private lateinit var formViewModelSubmissionService: FormViewModelSubmissionService
+    private lateinit var camundaTask: CamundaTask
+    private lateinit var processAuthorizationService: ProcessAuthorizationService
+    private lateinit var formViewModelService: FormViewModelService
+
+    @BeforeEach
+    fun setUp() {
+        objectMapper = MapperSingleton.get()
+        viewModelLoaderFactory = mock()
```

Comment:
Most of this setup can be eliminated by using constructor injection and `@Mock`.

Reply:
Yes, mock() is the equivalent of @Mock. :)

Reply:
You're mocking me, but the annotation version would cut your setup line size in half, getting the reader faster to the part that is important.